gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -156,8 +172,22 @@ def step(self, action, verbose=True):
                     inventory_changed_str = ''
                 print('{}: {}. {}'.format(
                     action_str, interaction_obj['objectType'], inventory_changed_str))
+        elif action_str.startswith('Rotate'):
+            if self.continuous_movement:
+                # Rotate actions
+                if action_str.endswith('Left'):
+                    self.absolute_rotation -= self.rotation_amount
+                    self.event = self.controller.step(
+                        dict(action='Rotate', rotation=self.absolute_rotation))
+                elif action_str.endswith('Right'):
+                    self.absolute_rotation += self.rotation_amount
+                    self.event = self.controller.step(
+                        dict(action='Rotate', rotation=self.absolute_rotation))
```

Comment:
Are you sure it is 'Rotate' and not 'Look'? 
```# you can turn in any direction by using the following command
# keep in mind that this is the absolution rotation of the agent in world space, so running the command a second time will not rotate the agent, it will just set the rotation to 275 degrees about the Y axis
controller.step(dict(action='Rotate', rotation=275.0)) 

# you can also controller where the agent looks in any amount, as with Rotate, this is an absolute value 
controller.step(dict(action='Look', horizon=37.0))
```
From https://github.com/allenai/ai2thor/issues/40

Reply:
Look is what it says on the tin. You can look arbitrarily anywhere. Rotate only rotates

Reply:
What is this then `so running the command a second time will not rotate the agent, it will just set the rotation to 275 degrees about the Y axis`

Reply:
both are absolute and can be made relative by storing the absolute rotation.
I don't see how your comment is a reply to what I said?
What's the issue, rotate is better?